theano/tensor/nnet/nnet.py

```diff
@@ -384,6 +384,7 @@ def c_code(self, node, name, inp, out, sub):
             }
         }
 
+        #pragma omp parallel for schedule(static, 32)
```

Comment:
Hi nouiz,

I have make softmax and softmaxGrad inherit from gof.OpenMPOp and commit the code.
Please help review it again.

Thanks very much.